### PR TITLE
fix: pull README.md into CIPs dir, fix CC0 links, add dropdown for CIPs

### LINKED ISSUE #176: Feature Request: move README.md out of root
# Problem

The relative links on the README.md on github do not work anymore.

Screenshot 2024-07-22 at 11 40 55 AM

# Solution

Move the majority of the README.md to another directory, so that it is obvious to use the site to navigate CIPs instead of the github repo

### LINKED ISSUE #177: bug: CC0 link broken
# Problem
Link to CC0 is broken on all CIPs pages on deployed site.

Screenshot 2024-07-22 at 12 01 50 PM

Screenshot 2024-07-22 at 12 01 45 PM

# Solution
Fix the relative links

### LINKED ISSUE #178: Feature Request: add toggle dropdown for CIPs section of menu so it does not fill entire sidebar
image

### PR DESCRIPTION
**Pull request by jcstein**
## Overview

Resolves #176 
Resolves #177 
Resolves #178 

NOTE: links that are in format of *.html are so that the site works. Known issue that they don't work on Github natively. But the site works.

https://github.com/jcstein/CIPs/blob/550a240fd01c0a80a7134eae9bc5fe9e487014c1/cips/README.md?plain=1#L5
https://github.com/jcstein/CIPs/blob/550a240fd01c0a80a7134eae9bc5fe9e487014c1/cips/wgs/README.md?plain=1#L6-L10

### CODE REVIEW

**Comment by jcstein** [comment]
it would also be possible to rename LICENSE to LICENSE.md and it should work, but when I renamed it, and tried to link it from (./LICENSE.md) or (../LICENSE.md) or (./../LICENSE.md) it failed similarly to the way I hit issues with:

> NOTE: links that are in format of *.html are so that the site works. Known issue that they don't work on Github natively. But the site works.
> 
> https://github.com/jcstein/CIPs/blob/550a240fd01c0a80a7134eae9bc5fe9e487014c1/cips/README.md?plain=1#L5
> https://github.com/jcstein/CIPs/blob/550a240fd01c0a80a7134eae9bc5fe9e487014c1/cips/wgs/README.md?plain=1#L6-L10

File: cips/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Celestia Improvement Proposal (CIP) process
+
+Read [CIP-1](./cip-1.md) for information on the CIP process.
+
+See the [WGs directory](./wgs/index.html) for Working Groups within the CIP process.
```
Comment (rootulp) [inline cips/README.md:5]: [optional] we could move this link to the root level README

**Review by rootulp — APPROVED** [review]

File: cips/README.md (hunk 1 above)
Comment (jcstein) [reply cips/README.md:5]: good idea. on the site, do you think that it is clear enough from the sidebar, that the WG category is there?

Comment (jcstein) [reply cips/README.md]: https://github.com/celestiaorg/CIPs/pull/179/commits/4c14da615fb8d76166236d564216f82c3f4934f6

it seems less out of place when it's in the root README, thank you

**Pull request merged**